src/components/UnorderedList/UnorderedList.stories.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+/**
+ * @file Unordered list stories.
+ * @copyright IBM Security 2019
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { components } from '../../../.storybook';
+
+import ListItem from '../ListItem';
+import { ListItem, UnorderedList } from '../../';
```

Comment:
- [ ] No need for a trailing slash here! 👋

```suggestion
import { ListItem, UnorderedList } from '../..';
```